managed/services/telemetry/config.default.yml

```diff
@@ -67,57 +59,189 @@ telemetry:
       - metric_name: "mongodb_version"
         label: "mongodb"
 
+  - id: PMMPostgresDBVersion
+    source: VM
+    query: pg_static{service_name="pmm-server-postgresql", service_type="postgresql"}
+    summary: "Monitored PostgreSQL services version"
+    data:
+      - metric_name: "pmm_server_postgresql_version"
+        label: "short_version"
+
   - id: PostgresDBVersion
     source: VM
-    query: pg_static{service_type="postgresql"}
+    query: pg_static{service_name!="pmm-server-postgresql", service_type="postgresql"}
     summary: "Monitored PostgreSQL services version"
     data:
       - metric_name: "postgresql_version"
         label: "short_version"
 
-#
+  #
   - id: GrafanaStatTotalUsers
     source: VM
     query: grafana_stat_total_users
     summary: "Total Grafana users"
     data:
-      - metric_name: "grafana_stat_total_users"
+      - metric_name: "pmm_server_grafana_stat_total_users"
         value: 1
 
-# PMM - Usage
+  # PMM - Usage
   - id: PMMUsageNodesCount
     source: PMMDB_SELECT
     query: count(*) AS nodes_count FROM nodes
```

Comment:
I was also thinking to add `-1` here, to exclude PMM itself.

Reply:
I think "WHERE" clause should be used for this

Reply:
let's do `Where` for now ... because  in the future we can have several components for PMM  (why not) and WHERE  will give better reminder about corner cases

Reply:
agree, that makes sense